ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/InMemoryPackageStore.scala

```diff
@@ -50,6 +52,15 @@ case class InMemoryPackageStore(
   def getLfPackageSync(packageId: PackageId): Option[Ast.Package] =
     packages.get(packageId)
 
+  override def lookupPackageUploadEntry(
+      submissionId: SubmissionId): Future[Option[domain.PackageUploadEntry]] = {
+    // TODO BH : not clear what codepath gets you here, but
+    // extending IndexPackagesService requires this method to be implemented
```

Comment:
There's actually no reason for this class to extend `IndexPackagesService`. Please remove that.

Reply:
woohoo!  that's what i was hoping for

Reply:
marcin has already removed -- marking as resolved